lib/vehicle.rb

```diff
@@ -22,4 +27,13 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def registered?    
+    !!plate_type && !!registration_date
+  end
+  
+  def new_registration_date
+    @registration_date = (Time.now)
+  end
```

Comment:
Since we're working with Dates, it would be good to use a `Date` object. Something like `Date.today` would work great here.